.gitignore

```diff
@@ -6,4 +6,4 @@ npm-debug.log
 coverage
 lib
 .DS_Store
-dist
+
```

Comment:
You added a ton of Probot boilerplate images and code by removing this. Are you sure?

Reply:
oh my bad. `dist` should be there for instant running. Did i forgot to revert it???